pgh_play/www/php/interface.php

```diff
@@ -201,5 +203,72 @@ function get_parks(){
 	return $results;
 }
 
+// enter new park into the db via new park form
+// this will store the following info in the db: Park name, address, rating, ages, comments
+//  all of these inputs are received by the new park form a user fills out
+// *****facilities and pictures still need to be implemented 
+function new_park(){
```

Comment:
Instead of taking a JSON object and decoding it into a php object   (e.g.  json_decode($_POST['obj'])), this seems less structured.   Also, it looks like this one function is doing multiple things, like taking a rating, setting fixed attributes, etc.